edi_oca/__manifest__.py

```diff
@@ -10,7 +10,7 @@
     basic automation and views for handling EDI exchanges.
     """,
     "version": "15.0.1.5.0",
-    "website": "https://github.com/OCA/edi",
+    "website": "https://github.com/OCA/edi-framework",
```

Comment:
better keep this PR for `edi_record_metadata_oca` only, can you please remove this change?

Reply:
What should I do now, if I don't change, it will break the [`pre-commit`](https://github.com/OCA/edi-framework/actions/runs/6320240070/job/17162466220?pr=18)?

Reply:
Ok, updated, thanks